browser/src/control/Control.NotebookbarBuilder.js

```diff
@@ -663,6 +665,25 @@ L.Control.NotebookbarBuilder = L.Control.JSDialogBuilder.extend({
 		return submenuOpts;
 	},
 
+	_getSlideSizeSubmenuOpts: function() {
```

Comment:
to remove

Reply:
we define menus in https://github.com/CollaboraOnline/online/blob/master/browser/src/control/jsdialog/Definitions.Menu.ts

which will be used by the regular "menubutton"

Reply:
Made the changes